Sources/StreamChatSwiftUI/ChatChannel/Reactions/MessageActions/DefaultMessageActions.swift

```diff
@@ -224,4 +249,74 @@ extension MessageAction {
         
         return flagMessage
     }
+    
+    private static func muteAction(
+        for message: ChatMessage,
+        channel: ChatChannel,
+        chatClient: ChatClient,
+        mutedUser: ChatUser,
+        onFinish: @escaping (MessageActionInfo) -> Void,
+        onError: @escaping (Error) -> Void
+    ) -> MessageAction {
+        let muteController = chatClient.userController(userId: mutedUser.id)
+        let muteAction = {
+            muteController.mute { error in
+                if let error = error {
+                    onError(error)
+                } else {
+                    onFinish(
+                        MessageActionInfo(
+                            message: message,
+                            identifier: "mute"
+                        )
+                    )
+                }
+            }
+        }
+        
+        let muteUser = MessageAction(
+            title: L10n.Message.Actions.userMute,
+            iconName: "speaker.slash",
+            action: muteAction,
+            confirmationPopup: nil,
+            isDestructive: false
+        )
+        
+        return muteUser
+    }
+    
+    private static func unmuteAction(
+        for message: ChatMessage,
+        channel: ChatChannel,
+        chatClient: ChatClient,
+        unmutedUser: ChatUser,
```

Comment:
Same naming idea here. Would prefer it to be called something like `userToUnmute`.